src/ucp/tag/eager_snd.c

```diff
@@ -319,10 +319,11 @@ void ucp_tag_eager_sync_send_ack(ucp_worker_h worker, void *hdr, uint16_t recv_f
     ucs_assert(reqhdr->req_id != UCP_REQUEST_ID_INVALID);
     req = ucp_proto_ssend_ack_request_alloc(worker, reqhdr->ep_id);
     if (req == NULL) {
-        ucs_fatal("could not allocate request");
+        /* drop the packet */
```

Comment:
please add error

Reply:
why needed? it is traced inside the function

Reply:
what function? This is memory allocation problem which is not expected in any flow, would be good to have an error

Reply:
https://github.com/openucx/ucx/blob/ec740b30c475ff1d2a15fa34686e6cb781db3b16/src/ucp/proto/proto_am.inl#L530


but we don't print any errors in case of EP wasn't found in the ptr map